dotcom-rendering/src/web/lib/getCommentContext.ts

```diff
@@ -101,9 +92,9 @@ export const getCommentContext = async (
 ): Promise<CommentContextType> => {
 	const url = joinUrl([ajaxUrl, 'comment', commentId.toString(), 'context']);
 	const filters = initFiltersFromLocalStorage();
-	const params = buildParams(filters);
+	const params = new URLSearchParams(buildParams(filters));
```

Comment:
This is a refactor, but I'm confident it does the same thing and we polyfill `URLSearchParams`.